src/medal.c

```diff
@@ -253,6 +266,35 @@ static void loadMedals()
 	}
 
 	free(buffer);
+
+#ifdef GAMERZILLA
+	Gamerzilla g;
+	char *gameSavePath = getGameSavePath();
+	char *gamerzillaPath = (char *)malloc(MAX_PATH_LENGTH);
+	strcpy(gamerzillaPath, gameSavePath);
+	strcat(gamerzillaPath, "gamerzilla/");
+	GamerzillaSetRead(&gamerzillaPakSize, &gamerzillaPakOpen, &gamerzillaPakRead, &gamerzillaPakClose);
+	GamerzillaStart(false, gamerzillaPath);
+	GamerzillaInitGame(&g);
+	g.version = 1;
+	g.short_name = strdup("edgar");
+	g.name = strdup("The Legend of Edgar");
+	g.image = strdup("gamerzilla/edgar.png");
+	for (i = 0; i < medalCount; i++)
+	{
+		if (medal[i].medalType == 0)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/bronze_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 1)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/silver_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 2)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/gold_medal.png", "gamerzilla/disabled_medal.png");
+		else if (medal[i].medalType == 3)
+			GamerzillaGameAddTrophy(&g, medal[i].name, medal[i].description, 0, "gamerzilla/ruby_medal.png", "gamerzilla/disabled_medal.png");
+	}
+	gameId = GamerzillaSetGame(&g);
+    GamerzillaClearGame(&g);
```

Comment:
Whi is this line indented differently?

Reply:
Must not have set the indent style properly when I added the line. Code has been fixed.